scripts/upload_image_dir.py

```diff
@@ -28,9 +28,9 @@ def main(date, auto_confirm=False, verbose=False):
     db = PanMongo()
     config = load_config()
     try:
-        pan_id = config['PAN_ID']
+        pan_id = config['pan_id']
```

Comment:
I wondered why you'd chosen the all caps, but added it to my pocs_local.yaml. I figured you had a reason.

Reply:
You asked me about it before and I switched it to lowercase at that time. (Originally I had it as an env var and had copied it in as such).